Kernel/Devices/DeviceManagement.cpp

```diff
@@ -109,6 +109,15 @@ void DeviceManagement::for_each(Function<void(Device&)> callback)
     });
 }
 
+ErrorOr<void> DeviceManagement::try_for_each(Function<ErrorOr<void>(Device&)> callback)
+{
+    return m_devices.with([&](auto& map) -> ErrorOr<void> {
+        for (auto& entry : map)
+            return callback(*entry.value);
```

Comment:
Shouldn't this be just a `TRY(callback(*entry.value))`, as is the enumeration will on process the first entry?

Reply:
Absolutely, good catch, fixed